python/pyspark/sql/connect/client.py

```diff
@@ -369,6 +370,22 @@ def register_udf(
     ) -> str:
```

Comment:
Since `register_udf` will be removed soon, I didn't make the workaround as a utility that can be reused or create a designated test file for that.

Reply:
Let's just don't fix this at all for now. We don't need to improve this that's going to be removed soon.

Reply:
Sounds good!